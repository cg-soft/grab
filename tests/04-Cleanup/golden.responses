Waiting for path/to/my/resource
GET http://host:port/path/to/my/resource?id=christian.goetze&op=grab
{
  "data": {
    "pos": {
      "/path": 1,
      "/path/to": 1,
      "/path/to/my": 1,
      "/path/to/my/resource": 1
    },
    "until": until
  },
  "id": "christian.goetze/path/to/my/resource",
  "op": "grab",
  "resource": "/path/to/my/resource",
  "status": "ok"
}
ok
Grabbed - Starting keepalive
Done waiting for path/to/my/resource
doing...
doing...
doing...
done.
Releasing path/to/my/resource
Keepalive log
GET http://host:port/path/to/my/resource?id=christian.goetze&op=grab
sleeping after getting response to keepalive:
{
  "data": {
    "pos": {
      "/path": 1,
      "/path/to": 1,
      "/path/to/my": 1,
      "/path/to/my/resource": 1
    },
    "until": until
  },
  "id": "christian.goetze/path/to/my/resource",
  "op": "grab",
  "resource": "/path/to/my/resource",
  "status": "ok"
}
GET http://host:port/path/to/my/resource?id=christian.goetze&op=grab
sleeping after getting response to keepalive:
{
  "data": {
    "pos": {
      "/path": 1,
      "/path/to": 1,
      "/path/to/my": 1,
      "/path/to/my/resource": 1
    },
    "until": until
  },
  "id": "christian.goetze/path/to/my/resource",
  "op": "grab",
  "resource": "/path/to/my/resource",
  "status": "ok"
}
GET http://host:port/path/to/my/resource?id=christian.goetze&op=release
{
  "data": 4,
  "id": "christian.goetze/path/to/my/resource",
  "op": "release",
  "resource": "/path/to/my/resource",
  "status": "ok"
}
Running cleanup command
ok
Released
Done releasing path/to/my/resource
Waiting for path/to/my/resource
GET http://host:port/path/to/my/resource?id=christian.goetze&op=grab
{
  "data": {
    "pos": {
      "/path": 1,
      "/path/to": 1,
      "/path/to/my": 1,
      "/path/to/my/resource": 1
    },
    "until": until
  },
  "id": "christian.goetze/path/to/my/resource",
  "op": "grab",
  "resource": "/path/to/my/resource",
  "status": "ok"
}
ok
Grabbed - Starting keepalive
Done waiting for path/to/my/resource
doing...
doing...
doing...
