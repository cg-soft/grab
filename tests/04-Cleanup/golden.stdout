Server running at http://host:port
GET: op=grab; id=christian.goetze; until=timestamp; resource=/path/to/my/resource
Adding christian.goetze/path/to/my/resource to end of /path/to/my/resource
Adding christian.goetze/path/to/my/resource to end of /path/to/my
Adding christian.goetze/path/to/my/resource to end of /path/to
Adding christian.goetze/path/to/my/resource to end of /path
GET: op=grab; id=christian.goetze; until=timestamp; resource=/path/to/my/resource
Found christian.goetze/path/to/my/resource at /path/to/my/resource[0]
Found christian.goetze/path/to/my/resource at /path/to/my[0]
Found christian.goetze/path/to/my/resource at /path/to[0]
Found christian.goetze/path/to/my/resource at /path[0]
GET: op=grab; id=christian.goetze; until=timestamp; resource=/path/to/my/resource
Found christian.goetze/path/to/my/resource at /path/to/my/resource[0]
Found christian.goetze/path/to/my/resource at /path/to/my[0]
Found christian.goetze/path/to/my/resource at /path/to[0]
Found christian.goetze/path/to/my/resource at /path[0]
GET: op=release; id=christian.goetze; until=timestamp; resource=/path/to/my/resource
Found christian.goetze/path/to/my/resource at /path/to/my/resource[0]
Removing christian.goetze/path/to/my/resource from /path/to/my/resource
Found christian.goetze/path/to/my/resource at /path/to/my[0]
Removing christian.goetze/path/to/my/resource from /path/to/my
Found christian.goetze/path/to/my/resource at /path/to[0]
Removing christian.goetze/path/to/my/resource from /path/to
Found christian.goetze/path/to/my/resource at /path[0]
Removing christian.goetze/path/to/my/resource from /path
GET: op=grab; id=christian.goetze; until=timestamp; resource=/path/to/my/resource
Adding christian.goetze/path/to/my/resource to end of /path/to/my/resource
Adding christian.goetze/path/to/my/resource to end of /path/to/my
Adding christian.goetze/path/to/my/resource to end of /path/to
Adding christian.goetze/path/to/my/resource to end of /path
GET: op=grab; id=christian.goetze; until=timestamp; resource=/path/to/my/resource
Found christian.goetze/path/to/my/resource at /path/to/my/resource[0]
Found christian.goetze/path/to/my/resource at /path/to/my[0]
Found christian.goetze/path/to/my/resource at /path/to[0]
Found christian.goetze/path/to/my/resource at /path[0]
GET: op=grab; id=christian.goetze; until=timestamp; resource=/path/to/my/resource
Found christian.goetze/path/to/my/resource at /path/to/my/resource[0]
Found christian.goetze/path/to/my/resource at /path/to/my[0]
Found christian.goetze/path/to/my/resource at /path/to[0]
Found christian.goetze/path/to/my/resource at /path[0]
